flowapi/flowapi/query_endpoints.py

```diff
@@ -409,3 +413,202 @@ async def get_available_dates():
+        schema:
+          type: string
+          format: date
+      responses:
+        '200':
+          description: Dates available for each event type.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  qa_checks:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        outcome:
+                          type: string
+                        type_of_query_or_check:
+                          type: string
+                        cdr_date:
+                          type: string
+                          format: date
+        '400':
+          description: Bad request
+        '401':
+          description: Unauthorized.
+        '404':
+          description: No QA checks of type specified found on date
+        '500':
+          description: Server error.
+      summary: Returns QA values for a given cdr and check type on a given date
+    """
+    current_user.can_get_qa()
+    return await get_qa_checks(cdr_type, check_id, check_date, check_date)
+
+
+async def get_qa_checks(cdr_type, check_id, start_date, end_date):
+    try:
+        dt.datetime.strptime(start_date, "%Y-%m-%d")
+        dt.datetime.strptime(end_date, "%Y-%m-%d")
+    except ValueError:
+        return (
+            dict(
+                status="bad request",
+                msg=f"Could not parse {start_date} or {end_date}; expected for format is YYYY-MM-DD",
+            ),
+            400,
+        )
+    params_dict = dict(
+        start_date=start_date,
+        end_date=end_date,
+        check_id=check_id,
+        cdr_type=cdr_type,
+    )
+    current_app.query_run_logger.info("get_qa_check", params_dict)
+    request.socket.send_json(
+        dict(request_id=request.request_id, action="get_qa_check", params=params_dict)
+    )
+    reply = await request.socket.recv_json()
+    if reply["status"] == "success":
+        if len(reply["payload"]["qa_checks"]) == 0:
+            return {
+                "status": "not found",
+                "msg": f"No qa checks found for {cdr_type}, {check_id} between {start_date} and {end_date}",
+            }, 404
+        else:
+            return {"qa_checks": reply["payload"]}, 200
+    else:
+        assert reply["status"] == "error"
+        return {"status": "error", "msg": reply["msg"]}, 500
+
+
+@blueprint.route("/qa", methods=["GET"])
+@jwt_required
+async def list_qa_checks():
+    """
+    Lists available QA checkons
+    ---
+    get:
+      responses:
+        '200':
+          description: Dates available for each event type.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  available_qa_checks:
+                    type: array
+                    items:
+                      type: string
```

Comment:
```suggestion
                      type: object
                      properties:
                        cdr_type:
                          type: string
                        type_of_query_or_check:
                          type: string
```

Or something like that I think